sdk/src/main/java/com/karhoo/sdk/api/network/request/AdyenPaymentMethodsRequest.kt

```diff
@@ -0,0 +1,9 @@
+package com.karhoo.sdk.api.network.request
+
+import android.os.Parcelable
+import com.google.gson.annotations.SerializedName
+import kotlinx.android.parcel.Parcelize
+
+@Parcelize
+data class AdyenPaymentMethodsRequest(@SerializedName("channel") val string: String = "Android")
```

Comment:
Should we call it `AdyenPaymentMethodsRequest` or just `PaymentMethodsRequest` ... it is part of Payments V3 API .... which would enable Adyen integration as a result.

Reply:
The endpoint is /v3/payments/adyen/payments-methods so I think probably AdyenPaymentMethodsRequest? Although I guess as it's just a channel maybe PaymentMethods should be okay.

Reply:
Actually, the iOS classes are all prepended with Adyen. Guess it would be worth keeping the naming in line on both platforms.

Reply:
fair enough :)

Reply:
I think it might not be a bad idea to differentiate the braintree related ones.  e.g. `NONCE_METHOD = "/v2/payments/payment-methods/braintree/get-payment-method`